tests/dialects/test_snowflake.py

```diff
@@ -89,6 +89,8 @@ def test_snowflake(self):
         self.validate_identity("SELECT TRANSLATE(column_name, 'abc', '123')")
         self.validate_identity("SELECT UNICODE(column_name)")
         self.validate_identity("SELECT WIDTH_BUCKET(col, 0, 100, 10)")
+        self.validate_identity("SELECT GROUPING(region)")
+        self.validate_identity("SELECT GROUPING(region, product)")
```

Comment:
No need for these– if you search for `GROUPING(` you'll find a few other tests that cover Snowflake.

```suggestion
```